localisation/00_zuwc_united_localisation.csv

```diff
@@ -272,21 +272,21 @@ EVTOPTB_WCBDP_80_alliance;One-eyed giants? As if ogres were not enough!;;Ein
 EVTOPTB_WCBDP_80_tooltip;You reject the union, but jeopardize your victory over the Aspect.\n;;Ihr lehnt das B�ndnis ab, verringert damit aber Eure Chancen auf einen Sieg �ber den Drachen.\n;;;;;;;;;;;x
 EVTDESC_WCBDP_85_case_01;We managed to inflict a crushing defeat on Deathwing and his brood! Alas, the great dragon himself escaped justice at the last moment. It is terrible to imagine that he could raise a new brood in the future.;;Wir waren in der Lage, Todesschwinge und seiner Brut eine vernichtende Niederlage beizubringen! Leider entkam der Gro�drache selbst seiner gerechten Strafe im allerletzten Augenblick. Schrecklich, sich vorzustellen, wie er in der Zukunft eine neue Brut heranzieht.;;;;;;;;;;;x
 EVTDESC_WCBDP_85_case_02;We managed to get rid of Deathwing, but at what cost... In addition, it seems the mate of the Aspect of Death managed to hide many nests around the Mountains, and we will never be able to find them all. Alas, now we must focus on Ner'zhul.;;Wir haben es geschafft, Todesschwinge zu vertreiben, aber um welchen Preis... Au�erdem scheint es, dass die Gef�hrtin des Aspekts des Todes es geschafft hat, viele Nester in den Klingenrandbergen abzulegen, und wir werden niemals alle finden k�nnen. Wie auch immer, jetzt m�ssen wir uns auf die Orks konzentrieren.;;;;;;;;;;;x
-EVTDESC_WCBDP_85_case_03;We seem to have underestimated their strength. This battle cost us too much, and the impact on the enemy was negligible. Deathwing is clearly able to gain a foothold in the Blade's Edge Mountains, and we cannot stand up to him right now. Better to leave it and focus on Ner'zhul...;;Es scheint, dass wir ihre St�rke untersch�tzt haben. Dieser Kampf kostete uns zu viel, und die Auswirkungen waren gering. Todesschwinge wird m�helos in der Lage sein, in den Klingenrandbergen Fu� zu fassen und jetzt sind wir nicht in der Lage, ihn zu konfrontieren. Es ist besser, sich abzuwenden und auf die Orks zu konzentrieren...;;;;;;;;;;;x
+EVTDESC_WCBDP_85_case_03;We seem to have underestimated their strength. This battle cost us too much, and the impact on the enemy was negligible. Deathwing was clearly able to gain a foothold in the Blade's Edge Mountains, and we cannot stand up to him right now. Better to leave it and focus on Ner'zhul...;;Es scheint, dass wir ihre St�rke untersch�tzt haben. Dieser Kampf kostete uns zu viel, und die Auswirkungen waren gering. Todesschwinge war m�helos in der Lage sein, in den Klingenrandbergen Fu� zu fassen und jetzt sind wir nicht in der Lage, ihn zu konfrontieren. Es ist besser, sich abzuwenden und auf die Orks zu konzentrieren...;;;;;;;;;;;x
```

Comment:
English suggestions
```suggestion
EVTDESC_WCBDP_85_case_03;We seem to have underestimated their strength. This battle has costed us far too much, and our impact on the enemy's forces has been negligible. Deathwing has been able to gain a foothold in the Blade's Edge Mountains, and we simply cannot stand up to his strength right now. It may be better to leave him undisturbed and instead focus on Ner'zhul...;;Es scheint, dass wir ihre St�rke untersch�tzt haben. Dieser Kampf kostete uns zu viel, und die Auswirkungen waren gering. Todesschwinge war m�helos in der Lage sein, in den Klingenrandbergen Fu� zu fassen und jetzt sind wir nicht in der Lage, ihn zu konfrontieren. Es ist besser, sich abzuwenden und auf die Orks zu konzentrieren...;;;;;;;;;;;x
```